File: JavaScript/Easy/FizzBuzz/FizzBuzz.js
const readLine = require("readline");

const rl = readLine.createInterface({input: process.stdin, output:process.stdout});

rl.on('line', (line) => {
    var input = line.split(' ');
    var divOne = parseInt(input[0])
    var divTwo = parseInt(input[1])
    var range = parseInt(input[2])

    for (let i = 1; i <= range; i++) {
        if (i%divOne == 0 && i%divTwo == 0 ) console.log("FizzBuzz");
        else if (i%divOne == 0 ) console.log("Fizz");
        else if (i%divTwo == 0 ) console.log("Buzz");
        else console.log(i);
        
    }

    
});

File: Kotlin/Easy/FizzBuzz/FizzBuzz.kt
fun main ()
{
    var line = readLine()

    while(line != null)
    {
        val (divOne, divTwo, range) = line.split(' ')
        var x = 1;

        while(x <= range.toLong())
        {
            if(x%divOne.toInt() == 0 && x%divTwo.toInt() == 0) println("FizzBuzz");
            else if(x%divOne.toInt() == 0) println("Fizz");
            else if(x%divTwo.toInt() == 0) println("Buzz");
            else println(x)
            x++

        }
 
        
        line = readLine()
    }
}